docs/resources/oidc_identity_mapping.md

```diff
@@ -70,15 +126,14 @@ resource "platform_oidc_identity_mapping" "my-github-oidc-group-identity-mapping
 <a id="nestedatt--token_spec"></a>
 ### Nested Schema for `token_spec`
 
-Required:
-
-- `scope` (String) Scope of the token. Must start with `applied-permissions/user`, `applied-permissions/admin`, `applied-permissions/roles:`, or `applied-permissions/groups:`. Group names must be comma-separated, double quotes wrapped, e.g. `applied-permissions/groups:\"readers\",\"my-group\",` Role permissions must be comma-separated, double quotes wrapped, e.g. `applied-permissions:roles:"Developer","Viewer". `username` is also required when setting role permission.
-
 Optional:
 
 - `audience` (String) Sets of (space separated) the JFrog services to which the mapping applies. Default value is `*@*`, which applies to all services.
 - `expires_in` (Number) Token expiry time in seconds. Default value is 60.
+- `groups_pattern` (String) Provide a pattern which is used to map OIDC groups to Artifactory groups.
```

Comment:
Should you add an example here? It should be inside of `{{}}`

Reply:
I added example in the section above.